content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -667,25 +667,16 @@ npx prisma db push --schema=/tmp/schema.prisma
 
 ### <inlinecode>db seed</inlinecode>
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-`db push` uses the Migration Engine and does not currently support the [MongoDB connector](./../../concepts/database-connectors/mongodb).
+`db seed` changed from Preview to Generally Available (GA) in 3.0.1.
```

Comment:
Super uncommon wording for the docs. How do we usually do this @molebox?

Reply:
It's a copy paste from migrate one